lib/base_service.js

```diff
@@ -35,163 +35,164 @@ class BaseService {
     }
 
     stop() {
-        if (this._ratelimiter) {
+        if ( this._ratelimiter ) {
             this._ratelimiter.stop();
         }
     }
 
-    _setAppBasePath(config) {
-        if (process.env.APP_BASE_PATH) {
+    _setAppBasePath( config ) {
+        if ( process.env.APP_BASE_PATH ) {
             this._basePath = process.env.APP_BASE_PATH;
-        } else if (config.app_base_path) {
+        } else if ( config.app_base_path ) {
             this._basePath = config.app_base_path;
-        } else if (/\/node_modules\/service-runner\/lib$/.test(__dirname)) {
+        } else if ( /\/node_modules\/service-runner\/lib$/.test( __dirname ) ) {
             // Default to guessing the base path
-            this._basePath = path.resolve(`${__dirname}/../../../`);
+            this._basePath = path.resolve( `${__dirname}/../../../` );
         } else {
-            this._basePath = path.resolve('./');
+            this._basePath = path.resolve( './' );
         }
     }
 
     _setUpDNSCaching() {
         const cacheConfig = this.config.dns_cache;
-        if (cacheConfig === false) {
+        if ( cacheConfig === false ) {
             return; // Caching disabled.
         }
 
-        require('dnscache')({
+        require( 'dnscache' )( {
             enable: true,
             ttl: cacheConfig && cacheConfig.ttl || 5,
             cachesize: cacheConfig && cacheConfig.size || 100
-        });
+        } );
     }
 
-    _getConfigUpdateAction(conf) {
-        throw new Error('_getConfigUpdateAction must be overwritten!');
+    // eslint-disable-next-line no-unused-vars
+    _getConfigUpdateAction( conf ) {
+        throw new Error( '_getConfigUpdateAction must be overwritten!' );
     }
 
-    start(conf) {
-        return this._getConfigUpdateAction(conf)
-        .then(() => {
-            const config = this.config;
+    start( conf ) {
+        return this._getConfigUpdateAction( conf )
+            .then( () => {
```

Comment:
Nonono... We line them on the same level everywhere else, this should be an override in the rules for sure.